core/src/wallet/common/api_impl/OperationApi.cpp

```diff
@@ -70,7 +68,7 @@ namespace ledger {
             return _backend.recipients;
         }
 
-        Operation &OperationApi::getBackend() {
+        ledger::core::Operation &OperationApi::getBackend() {
```

Comment:
We’re in the same namespace; why is that needed?

Reply:
It was just a 'clean up' to comply with declaration ! (and we can't get rid of the one in declaration because of some namespaces clash 😞)